tachyon/zk/lookup/BUILD.bazel

```diff
@@ -44,10 +44,7 @@ tachyon_cc_library(
 tachyon_cc_library(
     name = "verifier",
     hdrs = ["verifier.h"],
-    deps = [
-        ":type",
-        ":verifier_traits_forward",
-    ],
+    deps = [":verifier_traits_forward"],
```

Comment:
For this change, I think type just should not have been included from the previous commit.